### PR TITLE
refactor cli

### PR DESCRIPTION
**Pull request by mamachanko**
* distributes commands over individual modules

### CODE REVIEW

**Comment by dushyant88** [comment]
I like it. 

Just a suggestion - maybe you can import all the commands in `lymph/cli/__init__.py` and then you can just import all the commands from `from lymph import cli` and you can have `cli.XYZCommand` if you want to extend it in your service.

**Comment by dushyant88** [comment]
@mamachanko tests are failing otherwise +1

File: lymph/cli/inspect.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    def run(self):
+        address = self.args['<address>']
+        client = Client.from_config(self.config)
+        result = client.request(address, 'lymph.inspect', {}, timeout=5).body
+
+        print('RPC interface of {}\n'.format(self.terminal.bold(address)))
+
+        for method in sorted(result['methods'], key=lambda m: m['name']):
+            print(
+                "    {name}({params})\n\t {help}\n".format(
```
Comment (emulbreh) [inline lymph/cli/inspect.py]: can we stick to spaces here?

File: lymph/tests/integration/test_cli.py [hunk 2]
```diff
@@ -40,19 +41,19 @@ def test_negative_inspect(self):
...
-            discover          Show available services.
-            help              Display help information about lymph.
-            list              List available commands.
-            subscribe         Prints events to stdout.
-            instance          Run a single service instance (one process).
-            node              Run a node service that manages a group of processes on the same machine.
-            shell             Open an interactive Python shell locally or remotely.
-        """, config=False)
+        self.assert_lines_equal(['list'], u"""
+            {t.bold}tail           {t.normal}Stream the logs of one or more services.
```
Comment (mouadino) [inline lymph/tests/integration/test_cli.py]: how about other cli added  by other lymph extensions ?

**Comment by emulbreh** [comment]
+1

**Pull request merged**